conanfile.py

```diff
@@ -276,17 +300,12 @@ def build_with_autotools(self):
         with tools.environment_append(env_run.vars):
             with tools.chdir(self._source_subfolder):
                 use_win_bash = self.is_mingw and not tools.cross_building(self.settings)
-                autotools, autotools_vars = self._configure_autotools()
-
-                # autoreconf
-                self.run('./buildconf', win_bash=use_win_bash)
 
```

Comment:
Yeas, looks like it called twice by accident. 

Autoreconf must be run in any case.

```suggestion
                # autoreconf
                self.run('./buildconf', win_bash=use_win_bash)
```

Reply:
I believe you, I don't get autotool much.